admin-manual/customization/theming.rst

```diff
@@ -35,21 +36,136 @@ Customize how an AtoM theme looks
-   apps/qubit/templates/layout.php.
-
-2. Make css style changes to layout.css, graphic.css, form.css and menu.css
-
-3. Make changes to the templates in individual modules (found in the
-   apps/qubit/modules/xxx/templates directory)
-
-See
-`Symfony framework <http://symfony.com/legacy/doc/book/1_0/en/07-Inside-the-View-Layer>`_
-documentation for more assistance.
-
+Create a custom theme
+---------------------
+
+In most cases, it's probably enough to build a custom theme upon an existing
+one and preferably to use one that relies on our base theme Dominion, so you
+don't have to get your hands too dirty. It's in the details where most of the
+complexities are found. Dominion is the result of a cyclic process of testing
+and refining by a large community of users, try not to underestimate that!
+
+AtoM bundles two themes: arDominionPlugin and arArchivesCanadaPlugin. Their
+names follow the naming convention of Symfony 1.x plugins, because that is how
+themes are implemented in AtoM. You may want to read more about Symfony plugins
+later following `one of their guides <http://symfony.com/legacy/doc/gentle-introduction/1_4/en/17-Extending-Symfony#chapter_17_plug_ins>`_.
+
+arDominionPlugin is the default theme, i.e. the theme that will be used in a
+fresh installation. arArchivesCanadaPlugin was developed as an extension of the
+former and the following instructions will show you how to create your custom
+theme as we did with arArchivesCanadaPlugin.
+
+Assuming that you already have AtoM installed in your development environment
+(you can use our :ref:`Vagrant box <installation-vagrant>`), let's start
+building the plugin structure from the command line. Our theme is going to be
+called Corcovado (arCorcovadoPlugin). We are going to track its contents with
+git and publish them in a remote repository hosted by GitHub so we can enable
+others to contribute in the development. The repository is open source so you
+can use it for your own reference, see https://github.com/artefactual-labs/atom-theme-corcovado.
+You can also `create your own repository <https://help.github.com/articles/create-a-repo/>`_.
+
+Let's begin to do some real work:
+
+.. code-block:: bash
+
+   $ cd ~/atom/plugins
+   $ mkdir arCorcovadoPlugin
+   $ git init
+   $ echo "# Corcovado theme" > README.md
+   $ git add README.md
+   $ git commit -m "Initial commit"
+   $ git remote add origin git@github.com:artefactual-labs/atom-theme-corcovado.git
+   $ git push
+
+We've created an empty directory where our plugin is going to be contained,
+made a git repository of it, track our first file and publish our work to
+GitHub! However, our plugin does not meet the needed requirements for AtoM to
+recognize it and allow us to enable it. Let's make that happen:
+
+.. code-block:: bash
+
+   $ cd ~/atom/plugins/arCorcovadoPlugin
+   $ mkdir config
+   $ cd config
+
+Create a new file :file:`arCorcovadoPluginConfiguration.class.php` with the
+following contents:
+
+.. code-block:: php
```

Comment:
might be nice to have a comment or 2 in the code block here explaining what this does?